keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c

```diff
@@ -0,0 +1,191 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define EE_HANDS
```

Comment:
This line needs to be in `keyboards/keebio/levinson/keymaps/Rossman360_linux/config.h` to work.